src/EPR.RegulatorService.Facade.API/Controllers/OrganisationsSearchController.cs

```diff
@@ -174,7 +187,8 @@ public async Task<IActionResult> RemoveApprovedPerson(RemoveApprovedUsersRequest
         }
         catch (Exception e)
         {
-            _logger.LogError(e,"Error deleting approved user for organisation {organisationId}", request.OrganisationId);
```

Comment:
The standard way to use Microsoft logger in other project is as follows
_logger.LogError("Error deleting approved user for organisation {organisationId}", request.OrganisationId );